sdk/servicebus/azure-servicebus/azure/servicebus/_base_handler.py

```diff
@@ -477,37 +495,45 @@ def _mgmt_request_response(
         if keep_alive_associated_link:
             try:
                 application_properties = {
-                    ASSOCIATEDLINKPROPERTYNAME: self._handler._link.name  # pylint: disable=protected-access
+                    ASSOCIATEDLINKPROPERTYNAME: self._amqp_transport.get_handler_link_name(self._handler)
                 }
             except AttributeError:
                 pass
 
-        mgmt_msg = Message( # type: ignore # TODO: fix mypy error
-            value=message,
-            properties=Properties(reply_to=self._mgmt_target, **kwargs),
+        mgmt_msg = self._amqp_transport.create_mgmt_msg(
+            message=message,
             application_properties=application_properties,
+            config=self._config,
+            reply_to=self._mgmt_target,
+            **kwargs
         )
+
         try:
-            status, description, response = self._handler.mgmt_request(
+            return self._amqp_transport.mgmt_client_request(
+                self._handler,
                 mgmt_msg,
-                operation=amqp_string_value(mgmt_operation),
-                operation_type=amqp_string_value(MGMT_REQUEST_OP_TYPE_ENTITY_MGMT),
+                operation=mgmt_operation,
+                operation_type=MGMT_REQUEST_OP_TYPE_ENTITY_MGMT,
                 node=self._mgmt_target.encode(self._config.encoding),
-                timeout=timeout,  # TODO: check if this should be seconds * 1000 if timeout else None,
+                timeout=timeout,
```

Comment:
that comment above is something to keep in mind. Mostly for sockets its seconds, but if its going to be in an app property its ms

Reply:
I'll keep this comment then

Reply:
ah actually, this comment was moved to inside the _pyamqp_transport.py